server/versioninfo/feature.go

```diff
@@ -37,13 +36,22 @@ const (
 	// BatchSplit can speed up the region split.
 	// and PD will response the BatchSplit request.
 	BatchSplit
+	Version3_0
```

Comment:
Why do we need 3_0/4_0?

Reply:
This part of the code is split into another PR.